src/Initialization/init_ModelingToolkit.jl

```diff
@@ -4,8 +4,38 @@ eval(quote
                             simplify,
                             Operation
 
+   """
+       _vec(vars::NTuple{L, Union{<:Operation, <:Vector{Operation}}}) where {L}
+
+   Transform a tuple of operations into one vector of operations.
+
+   ### Input
+
+   - `vars` -- tuple where each element is either an `Operation` or a vector of operations
+
+   ### Output
+
+   A vector of `Operation` obtained by concatenating each tuple component.
+
+   ## Examples
+
+   ```julia
+   julia> vars = @variables x[1:2] y
+   (Operation[x₁, x₂], y)
+
+   julia> _vec(vars)
```

Comment:
I guess you need this...
```suggestion
   julia> LazySets._vec(vars)
```